tests/robustness/main_test.go

```diff
@@ -78,10 +78,6 @@ func testRobustness(ctx context.Context, t *testing.T, lg *zap.Logger, s testSce
 			t.Fatal(err)
 		}
 	}
-	err = failpoint.Validate(r.Cluster, s.failpoint)
```

Comment:
Removing validation as `PickRandom` should already limit failpoint picked to only ones passing validation. This leaves us with only failpoints that were hardcoded, which is ok as we only hardcode the reproduction cases like `make test-robustness-issue15271`.

Reply:
Question: Is it ever possible that a failpoint we hardcode is not `Available` for any of the cluster procs?

Reply:
Yes, for `make test-robustness-issue15271` it reproduces #15271 on older etcd version that didn't yet have  `SnapshotCatchupEntries` flag.